dietpi/dietpi-software

```diff
@@ -1347,6 +1347,16 @@ DietPi-Software will decrypt and use it for software installs. You can change it
 		aSOFTWARE_ONLINEDOC_URL[$software_id]='p=29377#p29377'
 		# - Bullseye
 		aSOFTWARE_AVAIL_G_DISTRO[$software_id,6]=0
+		#------------------
+		software_id=187
```

Comment:
ID 134 can be used now, that it became free one version before.

Reply:
done